layers/+lang/sql/packages.el

```diff
@@ -114,6 +121,8 @@
         ;; command around point, which is what you probably want.
         "sf" 'sql-send-paragraph
         "sF" 'spacemacs/sql-send-paragraph-and-focus
+        "sl" 'sql-send-line-and-next
```

Comment:
Since the user can configure `sql-pop-to-buffer-after-send-region`, shouldn't we make a wrapper for `sql-send-line-and-next` that doesn't focus the buffer no matter the value of `sql-pop-to-buffer-after-send-region`?

WDYT?

Reply:
If user sets `sql-pop-to-buffer-after-send-region` to `t` in his/her configurations, then `SPC m s l` will focus the buffer. So behaviour will not be different from `SPC m s L`.